pandas_gbq/gbq.py

```diff
@@ -781,6 +656,14 @@ def verify_schema(self, dataset_id, table_id, schema):
                                key=lambda x: x['name'])
         fields_local = sorted(schema['fields'], key=lambda x: x['name'])
 
+        # Ignore mode when comparing schemas.
+        for field in fields_local:
+            if 'mode' in field:
```

Comment:
not worth changing, but this could be marginally simpler as `field.pop('mode', None)`